src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -90,30 +80,32 @@ def create_dialog
       )
     end
 
+    def filesystem_items
+      [
+        Item(Id(Storage::FsType_EXT4), "Ext4"),
+        Item(Id(Storage::FsType_XFS), "XFS"),
+        Item(Id(Storage::FsType_BTRFS), "Btrfs"),
+        Item(Id(Storage::FsType_SWAP), "Swap"),
+        Item(Id(Storage::FsType_NTFS), "NTFS"),
+        Item(Id(Storage::FsType_VFAT), "VFAT")
+      ]
+    end
```

Comment:
Sure there are a lot of things we can do... but let's not overdo in this pull request.